packages/@aws-cdk/aws-eks/package.json

```diff
@@ -108,8 +108,8 @@
       "resource-attribute:@aws-cdk/aws-eks.Cluster.clusterSecurityGroupId",
       "resource-attribute:@aws-cdk/aws-eks.Cluster.clusterSecurityGroupId",
       "docs-public-apis:@aws-cdk/aws-eks.NodegroupAmiType.AL2_X86_64",
-      "docs-public-apis:@aws-cdk/aws-eks.NodegroupAmiType.AL2_X86_64_GPU"
-
+      "docs-public-apis:@aws-cdk/aws-eks.NodegroupAmiType.AL2_X86_64_GPU",
```

Comment:
Why do we need these two exclusions? Let’s just fix them

Reply:
Hi @eladb 

I got this error when I run `yarn awslint`

>error: [awslint:no-unused-type:@aws-cdk/aws-eks.LifecycleLabel] type or enum is not used by exported classes (declared at lib/cluster.ts:1001)

However, I was actually using this enum in the `Cluster` class.  
https://github.com/aws/aws-cdk/blob/9b515ab4f7b135f7431ef81b25ba690d317e69ee/packages/%40aws-cdk/aws-eks/lib/cluster.ts#L597

Can you advise how to fix this warning?

Reply:
It's used but not in a public API. Just remove the `export` keyword from the enum definition:

~`export`~`enum LifecycleLabel {`